Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -101,6 +101,12 @@ ul.treeViewRoot li[aria-selected=true]:focus > span.treeViewNode:hover
   box-shadow: var(--remotion-themed-box-shadow-focus-hover);
 }
 
+ul.treeViewRoot span.treeViewNodeHead,
+ul.treeViewRoot span.treeViewNodeHeadSelected
+{
+  flex-grow: 1;
+}
+
```

Comment:
Is this actually used now that we are using Javascript? The referenced spans already fill the relevant area since they highlight the hover-effect.